main.go

```diff
@@ -52,6 +54,8 @@ func (c *rootCmd) RegisterFlags(fl *flag.FlagSet) {
 	fl.BoolVar(&c.skipSync, "skipsync", false, "skip syncing local settings and extensions to remote host")
 	fl.BoolVar(&c.syncBack, "b", false, "sync extensions back on termination")
 	fl.BoolVar(&c.printVersion, "version", false, "print version information and exit")
+	fl.BoolVar(&c.noOpen, "no-open", false, "do not open web browser")
+	fl.StringVar(&c.bindFlag, "bind", "127.0.0.1", "local bind address for ssh tunnel")
```

Comment:
It's a little misleading to indicate that `127.0.0.1` is the default value of `--bind` since it will also pick a random port. I would leave the default value empty and indicate in the description that it will bind to `127.0.0.1` with a random port if not specified

Reply:
Oh, thought that I removed that. Will remove the default.